CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Changelog for `ta4j`, roughly following [keepachangelog.com](http://keepachangel
 - **MinusDMIndicator** moved to adx-package
 - `analysis/criterion`-package moved to root
 - `cost`-package moved to `analysis/cost`-package
+- **VersusBuyAndHoldCriterion** renamed to **`VersusEnterAndHoldCriterion`**
```

Comment:
I think this breaking change should not be part of this PR

Reply:
Ok. I did https://github.com/ta4j/ta4j/pull/885. Please merge https://github.com/ta4j/ta4j/pull/885 **before** this PR!